### ISSUE TITLE
Update benchmarks and use new version of other libraries

### ISSUE THREAD

**Issue by dehghani-mehdi** [scriban/scriban#367]
The current [benchmarks](https://github.com/scriban/scriban/blob/master/doc/benchmarks.md) is outdated and need to be updated. other libraries, specially [Fluid](https://github.com/sebastienros/fluid) got lots of changes since your benchmarks generated.

**Comment by xoofx** [comment]
Yep, just don't have time.

**Comment by xoofx** [comment]
Also, many features have been added recently that are making Scriban slower, while Fluid has been optimizing a lot in the meantime. So, a good pass at optimizing Scriban again would be worth it before doing any kind of new benchmarks...

**Comment by dehghani-mehdi** [comment]
> Also, many features have been added recently that are making Scriban slower, while Fluid has been optimizing a lot in the meantime. So, a good pass at optimizing Scriban again would be worth it before doing any kind of new benchmarks...

That's true, I ran couple of testcases and Fluid was faster than Scriban (~30%)
